tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -281,6 +282,13 @@ export const StripMenu = (props, context) => {
     return 'translucent';
   };
 
+  const disable_background_hover = (item) => {
+    if (item && item.cantstrip) {
+      return 'transparent';
```

Comment:
You can puy any color, but why you doesnt?

Reply:
I prefer it with no color